tests/conda_package_testfiles/test_conda_bear/TestBear.py

```diff
@@ -0,0 +1,21 @@
+
+from coalib.bearlib.abstractions.ExternalBearWrap import external_bear_wrap
+
+
+@external_bear_wrap(executable='the_test',
+                    settings={})
+class TestBear:
+    """
+    Just testing
+    """
+    LANGUAGES = {'TestLang'}
+    REQUIREMENTS = {'test1', 'test2'}
+    AUTHORS = {'TheTester'}
+    AUTHORS_EMAILS = {'testing@test.com'}
+    LICENSE = 'MIT'
```

Comment:
why would I be furious? We had a long disussion on MIT vs. AGPL and the result was that we weren't sure so we chose AGPL. Sadly that means that all code that doesn't run in another process (like linters) has to be AGPL too, legally